history.txt

```diff
@@ -1,6 +1,49 @@
 OMERO version history
 =====================
 
+5.4.0-m4 (August 2017)
+----------------------
+
+Developer preview release. Intended *only as a developer preview* for updating
+code before the full public release of 5.4.0. **Use at your own risk.**
+
+Changes include:
+
+- added method to JavaGateway to manipulate admin privileges
+- fixed graph policy rule to avoid stack overflow when moving large plates at once
+- fixed links to point to new website
+- fixed color convertion to rgba
+- fixed export as TIFF from webgateway
+- improved creating of rendering settings for image without any stats
+  e.g. 32bit images
+- fixed issue with jsonp decorator
+- fixed upload of script for admin with restricted privileges
+- fixed import bug in OMERO.insight
+- improved copy/paste of rendering setttings workflow in OMERO.web
+- fixed projection of images if the range of timepoint specified is not
+  the full range
+- added configuration to limit query that public user can do in OMERO.web
+- improved display of wells in OMERO.web
+- allowed admin with restricted privileges to create P/D for others in
+  OMERO.web
+- allowed the creation of admin with restricted privileges with password
```

Comment:
what's the 'with password' bit here for? It reads strangely.

Reply:
You could already create restricted admins, this adds a new method for setting a password for them when creating them.

Reply:
In that case:
"added support for setting a password for restricted administrators when creating them"